File: DKS-API/DTOs/SDevDtrFgtResult.cs
using System;
using DKS_API.Helpers;

namespace DKS_API.DTOs
{
    public class SDevDtrFgtResult : PaginationParams
    {
        public string article { get; set; }
        public string modelNo { get; set; }
        public string modelName { get; set; }
        
    }
}

File: DKS-SPA/src/app/core/_models/s-dev-dtr-fgt-result.ts
import { Pagination } from "./pagination";

export class SDevDtrFgtResult extends Pagination {

  article: string;
  modelNo: string;
  modelName: string;

  constructor() {

    super();
    this.isPaging = false; // 不開分頁
    this.article ="";
    this.modelNo ="";
    this.modelName ="";
  }
  
  public setPagination(pagination: Pagination) {

    this.currentPage = pagination.currentPage;
    this.itemsPerPage = pagination.itemsPerPage;
    this.totalItems = pagination.totalItems;
    this.totalPages = pagination.totalPages;
  }
}
